### PR TITLE
Nextcloud branding

### PR DESCRIPTION
**Pull request by irgendwie**
- [X] Adapt info.xml for the new Nextcloud app store
- [X] Add AGPL v3 license
- [x] Switch to [semver](http://semver.org/)
- [X] Replace all occurrences of ownCloud with Nextcloud
- [x] Replace favicon
- [x] Replace screenshot
- [x] New issue template
- [x] Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,11 +7,11 @@
 
 ![codecov.io](https://codecov.io/github/owncloud/contacts/branch.svg?branch=master)
 
-**A contacts app for [ownCloud](https://owncloud.org) and [Nextcloud](https://nextcloud.com).**
+**A contacts app for [Nextcloud](https://nextcloud.com) and [ownCloud](https://owncloud.org).**
```
Comment (MariusBluem) [inline README.md]: Should we totally remove **ownCloud** here as we have done this in calendar too 😁

**Comment by MariusBluem** [comment]
See my comments above 😜 I think we should use the README template used in calendar and mail too 😉 @jancborchardt ... I can jump in here for the README-part if you want 😎

**Comment by jancborchardt** [comment]
Readme template from the other apps sounds good! With the »Why is this so awesome« quick description section etc. :) Would be cool if you can do that @MariusBluem :)

**Comment by irgendwie** [comment]
@MariusBluem Would be nice, if you could also take a look at the issue template!

**Comment by phsc** [comment]
You should also add an AGPL-3.0 licence to the Nextcloud Contacts app.

**Comment by MariusBluem** [comment]
I've changed the issue template & moved it into the .github/ directory as we have also done with the server 😉

**Comment by tcitworld** [comment]
Ready ?

**Comment by MariusBluem** [comment]
README coming in some hours 😁 @tcitworld

**Comment by MariusBluem** [comment]
Ready to review 💪 ... Lets get this merged ASAP. Minor adjustments can be made later 😉 

cc @nextcloud/contacts

File: .github/issue_template.md [hunk 2]
```diff
@@ -71,9 +70,9 @@ Insert your config.php content here
 ```
 With access to your command line run e.g.:
 sudo -u www-data php occ ldap:show-config
-from within your ownCloud installation folder
+from within your Nextcloud installation folder
 
-Without access to your command line download the data/owncloud.db to your local
+Without access to your command line download the data/nextcloud.db to your local
```
Comment (tcitworld) [inline .github/issue_template.md]: It's still owncloud.db, right ?

File: README.md [hunk 3]
```diff
@@ -1,17 +1,26 @@
...
+![](https://raw.githubusercontent.com/nextcloud/screenshots/master/apps/Contacts/contacts.png)
 
-This is the new contacts app that has replaced the [old contacts app](https://github.com/owncloudarchive/contacts), and is only the frontend for the new server-integrated CardDAV-Backend.
+## Why is this so awesome?
 
-![](https://raw.githubusercontent.com/owncloud/screenshots/master/contacts/contacts.png)
+* :rocket: **Integration with other Nextcloud apps!** Currently Mail and Calendar – more to come.
+* :tada: **Never forget a birthday!** You can sync birthdays and other recurring events with your Nextcloud Calendar.
+* :busts_in_silhouette: **Sharing of Adressbooks!** You want to share your contacts with your friends or coworkers? No problem!
+* :see_no_evil: **We’re not reinventing the wheel!** Based on the great and open CardDAV backend.
```
Comment (tcitworld) [inline README.md]: You should say : based on the SabreDAV library.

File: README.md [hunk 4]
```diff
@@ -20,9 +29,7 @@ This is the new contacts app that has replaced the [old contacts app](https://gi
 - [Alexander Weidinger](https://github.com/irgendwie)
 - [Tom Needham](https://github.com/tomneedham)
 
-
-If you’d like to join, just go through the [issue list](https://github.com/owncloud/contacts/issues) and fix some. :)
-
+If you’d like to join, just go through the issue list and fix some. :)
```
Comment (tcitworld) [inline README.md]: Please leave and correct the link. :-)

**Comment by MariusBluem** [comment]
You were right 😁 ...missed `owncloud.db` 🙄

**Pull request merged**